.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 npm-debug.log
-build
```

Comment:
no, before publish in contracts actually I think
http://truffleframework.com/docs/getting_started/packages-npm